lib/iruby/command.rb

```diff
@@ -1,19 +1,34 @@
 require 'shellwords'
-require 'fileutils'
+require 'pathname'
```

Comment:
Whats the advantage of using pathname? I don't really see it and I would prefer to keep using fileutils.